compute-vars/medium.tfvars

```diff
@@ -0,0 +1,7 @@
+## Medium Configuration Template
+
+bastion                     = {memory      = "16",   processors  = "1",    "count"   = 1}
+bootstrap                   = {memory      = "32",   processors  = "0.5",  "count"   = 1}
+master                      = {memory      = "32",   processors  = "0.5",  "count"   = 3}
+worker                      = {memory      = "32",   processors  = "0.5",  "count"   = 2}
```

Comment:
To differentiate small from medium flavor lets have 3 worker nodes. 
I realise we will never be able to arrive at a consensus on good values for small/med/large. So this could be an acceptable compromise

Reply:
Changed worker count to 3.